.github/actions/chromeTestsSetup/action.yml

```diff
@@ -12,4 +12,4 @@ runs:
       gh-access-token: ${{ inputs.gh-access-token }}
   - name: Install chrome
     shell: 'bash'
-    run: npx @puppeteer/browsers install chrome@121
```

Comment:
do we want to keep repinning this? wondering if we should try to be @latest again.

Reply:
- yeah I originally changed it to that but some other errors happened and bruno pinned it. i am honestly down for either but doing @latest does require we update chromedriver regularly.
- takeaways when trying to fix tests were I think we merged something in that broke it. the only thing currently broken is the actual executing swap. Here is the error i was seeing:
<img width="554" alt="Screenshot 2024-09-17 at 1 34 18 AM" src="https://github.com/user-attachments/assets/9cb0caa6-19bf-40b9-a3f9-06e182ae66d4">
- my thought was since bx is moving slow and this release should get out we can unblock CI and then solve it afterwards